initfs/scripts/moslo-build.sh

```diff
@@ -108,17 +134,26 @@ add_dependencies()
                 fi
 
                 #Use ld-linux for libraries
+                LD_LINUX=$LD
+
                 $($LD --verify $i)
                 if [ "$?" -ne "2" ] ; then
-                        LD_LINUX=""
+                        #Get direct dependencies
+                        DIRECT_LIBS=$(objdump_find_lib_path $i)
+                        DEP="$DIRECT_LIBS"
+                        LD_LINUX="ldd"
+                        #Get full dependencies of the direct dependencies
+                        for lib in $DIRECT_LIBS; do
+                                TEMPDEPS=$($LD_LINUX $lib \
+                                        | sed -ne "s/.*[\t ]\(\/.*\) (.*/\1/gp")
```

Comment:
For me most sed and awk lines are completely incomprehensible no matter how you write it :P

Was the replacement of "/" with pipe "|" intentional?